tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 envlist=
-    py{38,39,310,311}-core
-    py{38,39,310,311}-lint
-    py{38,39,310,311}-wheel
-    py311-wheel-windows
+    py{38,39,310,311,312}-core
+    py{38,39,310,311,312}-lint
+    py{38,39,310,311,312}-wheel
+    windows-wheel
```

maybe:
```suggestion
    py{311,312}-windows-wheel
```
for consistency?

---

As it is, it just keeps circleci config drier. See .circleci/config.yml -> windows-wheel-steps -> windows-wheel-setup. If specified different TOXENVs, the whole windows-wheel-setup step would need to be duplicated for each job. Happy to make the change if you think it helps with clarity.

---

Nah, I think it's good as-is. ✨